static/bidder-info/programmaticX.yaml

```diff
@@ -0,0 +1,12 @@
+aliasOf: 'teqblaze'
+endpoint: "https://us-east.progrtb.com/pserver"
+maintainer:
+  email: "pxteam@programmaticx.ai"
+gvlVendorID: 1344
+userSync:
+  redirect:
+    url: "https://sync.progrtb.com/pbserver?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&ccpa={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&redirect={{.RedirectURL}}"
+    userMacro: "[UID]"
+  iframe:
+    url: "https://sync.progrtb.com/pbserverIframe?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&ccpa={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&pbserverUrl={{.RedirectURL}}"
+    userMacro: "[UID]"
```

Comment:
Verified iframe works with request to PBS setuid endpoint and programmaticX uid correctly set in the PBS cookie
<img width="1255" height="134" alt="Screenshot 2025-08-28 at 2 25 20 PM" src="https://github.com/user-attachments/assets/8ea6e69c-65de-4f44-913b-768e9f6880d0" />